docs/readthedocs/usersguide.md

```diff
@@ -25,6 +25,35 @@ You can find and download the source code of the latest stable SIMONA version [h
 
     $ git clone https://github.com/ie3-institute/simona.git
 
+## Docker
+
+Docker is a tool used to containerize the entire setup. A container is like a lightweight, isolated environment that bundles everything the software needs into a single package. It is the tool used to handle the entire project setup in a single space. 
+
+Using Docker, various applications across different platforms can be run simultaneously. Instead of installing different plugins required for simona on the local machine, docker can be used to run the entire simona setup after creating an image while interacting with different environments.
+Each component has its own environment inside the Docker container setup and are defined in a yml file to ensure consistency.
+
+SIMONA provides a Dockerfile in the root directory that can be used to build a Docker image and run simulations inside a container.
+
+### Building Docker Image
+
+1. Locate the dockerfile in the root directory from the SIMONA.
+2. Build the image by specifying the below command:
+    
+       docker build --build-arg version=2.1.0 --build-arg snapshotSuffix=-SNAPSHOT -t simona 
```

Comment:
Leave out `--build-arg snapshotSuffix=-SNAPSHOT`, please. Currently, jars built via shadowJar do not have such suffixes.